src/gui/owncloudgui.cpp

```diff
@@ -948,14 +1036,41 @@ void ownCloudGui::slotPauseAllFolders()
 
 void ownCloudGui::runNewAccountWizard()
 {
-    if (!_wizard) {
-        _wizard = new OwncloudSetupWizard(settingsDialog());
-        connect(_wizard, &OwncloudSetupWizard::ownCloudWizardDone, _wizard, &OwncloudSetupWizard::deleteLater);
-        connect(_wizard, &OwncloudSetupWizard::ownCloudWizardDone, ocApp(), &Application::slotownCloudWizardDone);
+    if (_wizardController.isNull()) {
+        // passing the settings dialog as parent makes sure the wizard will be shown above it
+        // as the settingsDialog's lifetime spans across the entire application but the dialog will live much shorter,
+        // we have to clean it up manually when finished() is emitted
+        _wizardController = new Wizard::SetupWizardController(settingsDialog());
+
+        // while the wizard is shown, new syncs are disabled
         FolderMan::instance()->setSyncEnabled(false);
```

Comment:
We should investigate why this is needed.

Reply:
Should I leave a TODO? Should we open an issue?

Reply:
Please open an issue.